tests/network-setups/fabric/dev/scripts/setupAppCC.sh

```diff
@@ -13,8 +13,8 @@ if [ -d "${CHAINCODE_PATH}/${CC_NAME}" ]; then
 fi
 cp -r ${APP_CC_PATH} ${CHAINCODE_PATH}/
 
-if [ "${CC_NAME}" = "simpleasset" ]; then
-  cp -r ${INTEROPCC_CHAINCODE_PATH}/fabric-interop-cc/interfaces/asset-mgmt ${CHAINCODE_PATH}/${CC_NAME}/
-fi
+# if [ "${CC_NAME}" = "simpleasset" ]; then
+#   cp -r ${INTEROPCC_CHAINCODE_PATH}/fabric-interop-cc/interfaces/asset-mgmt ${CHAINCODE_PATH}/${CC_NAME}/
+# fi
```

Comment:
Can we just delete these lines? Any reason to retain them?

Reply:
Deleted.